src/Aglais.h

```diff
@@ -0,0 +1,93 @@
+/* -*- mode: c++ -*-
```

Comment:
I do not like putting more files directly in `src/` to be honest.

We have the plugin headers there, for backwards compatibility, and for consistence. We could deprecate them, and ask users to include `kaleidoscope/plugin/$foo.h` instead, but that would not be consistent with how third-party addons are included, which will be `Kaleidoscope-$foo.h`, due to how Arduino finds libraries.

But this testing framework is not something our end-users will want to use and work with, except perhaps for running tests. It's more developer oriented, and doesn't have to be consistent with 3rd party plugins. So I'd move this header to `src/aglais/`, or perhaps even `src/kaleidoscope/aglais`, or `src/kaleidoscope/simulator/aglais` (with namespaces adjusted accordingly). Or, if we opt to name it according to function, `src/kaleidoscope/simulator/log_protocol` perhaps?